src/pages/Products/SelectFilter.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useRef, useState } from 'react';
+import css from './SelectFilter.module.scss';
+
+function SelectFilter(props) {
+  const [appear, setAppear] = useState(false);
+
+  const clickDropList = e => {
+    setAppear(!appear);
+  };
+
+  const outsideRef = useRef(null);
+
+  useEffect(() => {
+    function handleClickOutside(event) {
+      if (outsideRef.current && !outsideRef.current.contains(event.target)) {
+        setAppear(appear);
+      }
+    }
+    document.addEventListener('click', handleClickOutside);
+
+    return () => {
+      document.removeEventListener('click', handleClickOutside);
+    };
+  }, [outsideRef]);
+
+  const [selectFilter, setSelectFilter] = useState(false);
+
+  const clickChangeFilter = e => {
+    setSelectFilter(!selectFilter);
+  };
+
+  return (
+    <div className={css.select_filter} ref={outsideRef}>
+      <div
+        className={[css.drop_list, css.drop_list_title].join(' ')}
```

Comment:
```suggestion
        className={`${css.drop_list} ${css.drop_list_title}`}
```
join을 사용하지 않고 템플릿 리터럴을 사용하는 방법도 있는데
이 방법도 알고 계시면 좋을 것 같아요!

Reply:
네! 더 깔끔하고 직관적이네요 수정하도록 하겠습니다~ 리뷰 감사합니다!!!